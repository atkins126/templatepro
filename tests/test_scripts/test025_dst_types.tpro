DataSets Field Types
{{for t in testdst}}
field_ftString: {{:t.field_ftString}}
field_ftInteger: {{:t.field_ftInteger}}
field_ftFloat: {{:t.field_ftFloat}}
field_ftSingle: {{:t.field_ftSingle}}
field_ftFMTBcd: {{:t.field_ftFMTBcd}}
field_ftTimeStamp: {{:t.field_ftTimeStamp}}

{{endfor}}
