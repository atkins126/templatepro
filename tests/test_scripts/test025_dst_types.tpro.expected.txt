DataSets Field Types
field_ftString: Daniele Teti
field_ftInteger: 1
field_ftFloat: 123.456
field_ftSingle: 123.456001281738
field_ftFMTBcd: 1234.5678
field_ftTimeStamp: 1979-11-04 00:10:00

field_ftString: Bruce Banner
field_ftInteger: 2
field_ftFloat: 234.567
field_ftSingle: 234.567001342773
field_ftFMTBcd: 2345.5678
field_ftTimeStamp: 1979-11-04 00:10:00

